com.dynamo.cr/com.dynamo.cr.bob/src/com/defold/extension/pipeline/texture/TextureCompression.java

```diff
@@ -44,8 +45,8 @@ public static ITextureCompressor getCompressor(String name) {
         ITextureCompressor compressor = compressors.getOrDefault(name, null);
         if (compressor == null) {
             // There should always be a default compressor availale.
-            if (name.equals(TextureCompressorDefault.TextureCompressorName)) {
-                compressor = new TextureCompressorDefault();
+            if (name.equals(TextureCompressorUncompressed.TextureCompressorName)) {
+                compressor = new TextureCompressorUncompressed();
```

Comment:
Decided to rename the compressor to "uncompressed" instead, "default" is a bad name.